packages/trader/src/AppV2/Components/BottomNav/bottom-nav.scss

```diff
@@ -1,16 +1,15 @@
+$BOTTOM_NAV_HEIGHT: 56px;
```

Comment:
do we need to sure quill-token for this value . Not sure , just asking?

Reply:
really not sure if this height value has to come from token. In future if value of token is changed we don't want height of bottom nav to be changed.

Reply:
please feel free to use quill size token for this since they have been stable for a long time now, actually i don't remember them ever changing :)